blocks/library/button/index.js

```diff
@@ -219,7 +207,13 @@ export const settings = {
 	},
 
 	deprecated: [ {
-		attributes: blockAttributes,
+		attributes: {
+			...blockAttributes,
+			align: {
```

Comment:
Since a `deprecated` entry supports `supports`, could we just include `align` there?

Reply:
I don't think it will work. The `deprecated` entry doesn't copy `attributes`, `supports` and `save`: `...omit( blockType, [ 'attributes', 'save', 'supports' ] )`. Hooks we have with the current implementation don't modify the `deprecated` block types.  @youknowriad can you confirm if the current implementation is valid?